src/nuklear.h

```diff
@@ -5465,16 +5465,4 @@ template<typename T> struct nk_alignof{struct Big {T x; char c;}; enum {
 #define NK_ALIGNOF(t) ((char*)(&((struct {char c; t _h;}*)0)->_h) - (char*)0)
 #endif
 
-#ifdef NK_IMPLEMENTATION
```

Comment:
Moved from here into `src/nuklear_internal.h`.
These are internal implementation details that do not need to be exposed.